muxers/mplex/src/codec.rs

```diff
@@ -25,10 +25,10 @@ use core::Endpoint;
 use tokio_io::codec::{Decoder, Encoder};
 use unsigned_varint::{codec, encode};
 
-// Arbitrary maximum size for a packet.
+// Maximum size for a packet: 1Mb.
```

Comment:
Well, the idea is that we should mention that the value "1MB" (and not "1Mb") is part of the specifications. Otherwise the comment is tautological.